lib/router.js

```diff
@@ -73,4 +70,12 @@ Router.prototype.match = function r_match(method, url) {
   return matches;
 };
 
+Router.prototype._sanitizeMethod = function(method) {
+  var sanitizedMethod = method.toLowerCase().trim();
+  if (this.methods.indexOf(sanitizedMethod) === -1) {
+    throw new Error('Method "' + method + '" is not supported');
```

Comment:
This is the correct way to do it. I read it as a `console.error` so I was suggesting the parameter approach but I was wrong. Sorry.